lib/tasks/statistics/load_uk_grid_carbon_intensity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :statistics do
+  desc "Get benchmarks for loading uk grid carbon intensity data"
+  task load_uk_grid_carbon_intensity: :environment do
+    uk_grid_carbon_intensity_data = GridCarbonIntensity.new
```

Comment:
This should be within the benchmarked code.

Reply:
:+1: done